routes/led_with_ic/list.tsx

```diff
@@ -0,0 +1,185 @@
+import { Table } from "lib/ui/Table"
+import { withWinterSpec } from "lib/with-winter-spec"
+import { z } from "zod"
+import { formatPrice } from "lib/util/format-price"
+
+export default withWinterSpec({
```

Comment:
look at the other similar files, this file is not correct for the api pattern, it should be two lines